lib/DueDatePicker/DueDateHeader.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import cx from 'classnames';
+import DueDateTime from './DueDateTime';
+import DropdownMenu from '../DropdownMenu';
+import Icon from '../Icon';
+
+const DueDateHeader = props => {
+  if (props.dueDate) {
+    const date = moment(props.dueDate);
+
+    const day = date.calendar(null, {
+      sameDay: '[Today at]',
+      nextDay: '[Tomorrow at]',
+      nextWeek: 'dddd [at]',
+      lastDay: '[Yesterday at]',
+      lastWeek: '[Last] dddd [at]',
+      sameElse: 'MMMM Do YYYY [at]'
+    });
+
+    const time = props.dueTime.format('LT');
+
+    const classes = cx('duedate__header', {
+      'duedate__header--overdue': date < moment().set({ hour: 0, minute: 0 })
+    });
+
+    return (
+      <div className={classes}>
+        Due date
+        <div className="duedate__header--date">
+          {day} <DueDateTime time={time} setTime={props.setTime} />
+        </div>
+        <DropdownMenu
+          className="duedate__remove"
+          caret={false}
+          type="link"
+          shouldDisplay
+          listClassName="dropdown-menu-right"
+          items={[
+            {
+              action: props.removeDueDate,
+              name: 'Remove due date',
+              linkType: 'danger'
+            }
+          ]}
+        >
+          <Icon style={{ fontSize: '18px' }} name="menuDotted" />
```

Comment:
Should this line style be here?

Reply:
My bad not totally sure where I copied that from as I cant see it anywhere else lol